libuavcan/include/uavcan/node/abstract_node.hpp

```diff
@@ -109,7 +109,7 @@ class UAVCAN_EXPORT INode
      * @param flags             CAN IO flags. Please refer to the CAN driver API for details.
      */
     int injectTxFrame(const CanFrame& frame, MonotonicTime tx_deadline, uint8_t iface_mask,
-                      CanTxQueue::Qos qos = CanTxQueue::Volatile,
+                      Qos qos = Qos::Volatile,
```

Comment:
I don't think there is any point keeping QoS around since it's no longer useful. Let's drop it and update the IO manager API accordingly.